### DISCUSSION TITLE
Rock failure problem

### DISCUSSION THREAD

**Discussion by FURUNZ** [SeisSol/SeisSol#1014] [Q&A] [answered]
I did not see the setting of rock material strength in the seissol material parameter setting file. Can seissol input strength information to study whether the stress wave generated by the earthquake will reach the strength of rock mass.
I want to simulate the stress waves generated by the tensile failure process of rocks, but I do not know whether it can be achieved in seissol.

**Comment by Thomas-Ulrich** [comment, marked as answer]
There is only a visco-plasticity rheology in SeisSol, which transforms stress exceeding rock strength into plastic strain, without generating new waves (it just absorbs some of the waves' energy).

**Comment by FURUNZ** [comment]
Is it possible to apply the global tensile stress first and set the tensile stress suddenly to 0 in a region? How should this situation be implemented in fault.yaml? Will this radialize elastic waves




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "SeisSol/SeisSol"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年1月22日(星期一) 晚上8:54
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [SeisSol/SeisSol] Rock failure problem (Discussion #1014)





 
There is only a visco-plasticity rheology in SeisSol, which transforms stress exceeding rock strength into plastic strain, without generating new waves (it just absorbs some of the waves' energy).

**Reply by Thomas-Ulrich** [reply]
not possible.
